src/Model/TalkCommentMapper.php

```diff
@@ -259,10 +311,8 @@ public function save(array $data)
             throw new Exception("Duplicate comment");
         }
 
-        $sql = 'insert into talk_comments (talk_id, rating, comment, user_id, '
-               . 'source, date_made, private, active) '
-               . 'values (:talk_id, :rating, :comment, :user_id, :source, UNIX_TIMESTAMP(), '
-               . ':private, 1)';
+        $sql = 'insert into talk_comments (talk_id, rating, comment, user_id, source, private, active) '
```

Comment:
I would rather see the `created_at` being set here explicitly to not depend on the correct setting of the database-server. If we set it explicitly ourselves we know that the timezone-settings are correct (or can't blame the database-server if they aren't).

So the value should be `(new DateTimeImmutable())->setTimezone(new DateTimezone('UTC')->format('Y-m-d\TH:i:s')`